internal/commands/pull/inputs_test.go

```diff
@@ -48,6 +48,15 @@ func TestPullInputsResolve(t *testing.T) {
 			i := inputs{AppVersion: realm.AppConfigVersion20200603}
 			assert.Equal(t, errConfigVersionMismatch, i.Resolve(profile, nil))
 		})
+
+		t.Run("should return an error when include node modules and include package json are both provided", func(t *testing.T) {
+			i := inputs{IncludePackageJSON: true, IncludeNodeModules: true}
```

Comment:
[meganit/optional] I prefer to keep setup code (like `profile, teardown` and `defer`s) above creating the thing we're actually testing (`inputs` here)